src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/index.js

```diff
@@ -29,8 +29,9 @@ export default {
             user: [],
             isLoading: false,
             itemToDelete: null,
+            confirmDeleteModal: false,
+            confirmPasswordModal: false,
             disableRouteParams: true,
-            confirmPassword: '',
```

Comment:
This is a breaking change.

Reply:
So should I revert this and mark it as deprecated?

Reply:
Yes 😉